Src/CSharpier.Tests/TestFiles/SwitchExpression/SwitchExpressions.cst

```diff
@@ -18,5 +18,30 @@ class ClassName
             (DoorState.Locked, Action.Unlock, true) => DoorState.Closed,
             (var state, _, _) => state
         };
+
+        return someValue switch
+        {
+            SomeSimpleObject => DoSomething(),
+            { IsParameter: true } => 1,
+            // this comment shouldn't affect the next line
+            { IsParameter: false } => 0,
+            SomeLongObject someLongObject
+              => CallSomeMethodWith____________________________(someLongObject),
+            YetAnotherObject
+              => CallSomeMethod(
+                  someValue,
+                  andOtherParameters,
+                  thatMakeThisLongEnoughToBreak___________________
+              ),
+            OneMore
+              => "someStrings"
+              + "moreStrings"
```

Comment:
Is this a regression? Didn't we have binary operators (including string concats) indent?

Reply:
Right now the Node that contains the BinaryExpression sometimes indents. 
I did have this indenting, but then starting thinking BinaryExpression itself should indent, which means everything that may contain them doesn't have to. As part of #37 I plan on figuring all of this out. And I'm finally going to work on that today.